### PR TITLE
feat(js): React Native

### PR DESCRIPTION
**Pull request by blu3beri**
Builds on top of #88 

- React Native wrapper implementation
- Uses TurboModules for improved performance and sync behaviour
- Has not been used in production yet, but it will be integrated and tested in AFJ

@TimoGlastra Do you think it is smart if you review the TypeScript side?

### CODE REVIEW

File: wrappers/javascript/react-native/package.json [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+    "format": "yarn prettier --write",
+    "prettier": "prettier --ignore-path .gitignore '**/*+(.js|json|ts|md|yml|yaml)'",
+    "build": "yarn clean && yarn compile",
+    "clean": "rimraf -rf ./lib",
+    "compile": "tsc -p tsconfig.build.json",
+    "example": "yarn --cwd example",
+    "release": "release-it"
+  },
+  "dependencies": {
+    "indy-vdr-shared": "../shared/indy-vdr-shared-v103.0.0.tgz"
```
Comment (TimoGlastra) [inline wrappers/javascript/react-native/package.json:66]: this should not be a tar right?

File: wrappers/javascript/react-native/tsconfig.build.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{
+  "extends": "../tsconfig.build.json",
+  "compilerOptions": {
+    "outDir": "./build",
+    "skipLibCheck": true,
+    "types": ["react-native"]
+  },
+  "include": ["src/**/*"],
+  "exclude": ["example"]
```
Comment (TimoGlastra) [inline wrappers/javascript/react-native/tsconfig.build.json:9]: I think this is not needed anymore right as the example will be at the same level of the react-native / node packages right?

File: wrappers/javascript/shared/yarn.lock [hunk 3]
```diff
@@ -0,0 +1,449 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
Comment (TimoGlastra) [inline wrappers/javascript/shared/yarn.lock:1]: I think we only need one yarn.lock in the root `javascript` directory

File: yarn.lock [hunk 4]
```diff
@@ -0,0 +1,4 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
Comment (TimoGlastra) [inline yarn.lock:1]: I think this can be removed

**Review by TimoGlastra — APPROVED** [review]
Few small nits, but overall LGTM!

**Comment by andrewwhitehead** [comment]
I think we can close the other two PRs as this supersedes them?

Why is the name 'react-native-indy-vdr-react-native'?

**Comment by blu3beri** [comment]
> I think we can close the other two PRs as this supersedes them?

Yes, we can close the other pull requests. 


> Why is the name 'react-native-indy-vdr-react-native'?

That happened during the setup of the project. I will rename any reference to 'indy-vdr-react-native'.

**Pull request closed without merging**